.circleci/build_googletest.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
```

Comment:
I think the place of this shell script -> tests/ is better than .circleci/

Reply:
I'll move the script